pkg/apis/kubedirector/v1beta1/kubedirectorapp_types.go

```diff
@@ -34,6 +34,7 @@ type KubeDirectorAppSpec struct {
 	DefaultEventList    *[]string           `json:"defaultEventList,omitempty"`
 	Capabilities        []corev1.Capability `json:"capabilities,omitempty"`
 	SystemdRequired     bool                `json:"systemdRequired,omitempty"`
+	DefaultLogoURL      string              `json:"defaultLogoURL,omitempty"`
```

Comment:
[nit] wondering if we could simply name this field as `LogoURL`. The other `Default` fields like `DefaultImageRepoTag`, `DefaultSetupPackage` ... are also applicable at member level. Since logo would be a KDApp-specific attribute (like `Version`), could we also keep the logo field name consistent with this scheme?

Reply:
I think it makes sense. Will change it.